Library/Homebrew/test/support/helper/spec/integration_command.rb

```diff
@@ -0,0 +1,92 @@
+require "open3"
+
+RSpec::Matchers.define_negated_matcher :not_to_output, :output
+RSpec::Matchers.define_negated_matcher :be_a_failure, :be_a_success
+
+RSpec::Matchers.define :be_a_success do
```

Comment:
In that case, can we move it into a shared location (and other marchers created in spec files)? I don't like creating global state in a single spec file.